### ISSUE TITLE
Add usage support in IntelliJ IDEA 

### ISSUE THREAD

**Issue by hellnn** [aleksandr-m/gitflow-maven-plugin#307]
Add support IntelliJ IDEA using settings maven from its settings, for example using param -Dmaven.home

**Comment by aleksandr-m** [comment]
It is not clear. Can you be more specific?

**Comment by hellnn** [comment]
In most cases, when launching maven tasks, the built-in maven for IDEA is used, so it would be convenient to use a plugin with maven settings from IDEA

**Comment by aleksandr-m** [comment]
You can use `mvnExecutable` to use different maven. Is this enough?

**Comment by hellnn** [comment]
Yes, a can try, but I think, will be convenient if plugin will use -Maven.home variable when it is specified

**Comment by aleksandr-m** [comment]
@hellnn So this is only about using different maven executable for internal maven commands? Take a look at how plugin handles maven wrapper - [here](https://github.com/aleksandr-m/gitflow-maven-plugin/commit/72da5071405e18d8f969f94d2bc291ef2aeded0f). Is that what you want to handle for IDEA somehow?

**Comment by hellnn** [comment]
Yes, IDEA sets -Dmaven.home varable at executing internal maven goals

For example:

`"C:\Program Files\Java\jdk-11.0.6\bin\java.exe" -Dmaven.multiModuleProjectDirectory=C:\Users\hell\IdeaProjects\sp-dictionary "-Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\plugins\maven\lib\maven3" "-Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\plugins\maven\lib\maven3\bin\m2.conf" "-Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\plugins\maven\lib\maven-event-listener.jar" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\lib\idea_rt.jar=56192:C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\bin" -Dfile.encoding=UTF-8 -classpath "C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\plugins\maven\lib\maven3\boot\plexus-classworlds-2.6.0.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2021.2\plugins\maven\lib\maven3\boot\plexus-classworlds.license" org.codehaus.classworlds.Launcher -Didea.version=2021.2.1 clean`
